modules/eventing/pages/eventing-faq.adoc

```diff
@@ -1,58 +1,82 @@
 = Frequently Asked Questions
 
 [abstract]
-Some questions, frequently asked about the Eventing service and Functions.
+Some questions, frequently asked about the Eventing Service and Functions.``
 
-== Frequently Asked Questions
 
-* What languages are supported?
-+
-Javascript is the language to be used while creating Functions.
-Node modules cannot be imported: only simple Javascript can be used.
+== *Generic*
 
 * Is Eventing an MDS-enabled service?
+
 +
-Yes.
-MDS enables workload isolation in a Couchbase cluster.
+Yes. MDS enables workload isolation in a Couchbase cluster.
 Eventing nodes can be added as more Functions are added, or if the mutation rates are high.
 
+
+* What kind of nodes do I run my Eventing Service on?
++
+Eventing leverages the latest trends in multi-core CPUs; therefore nodes selected for Eventing should optimally have a higher number of cores than those for indexing.
+
+
+* Can the Eventing Service be co-located with other services?
++
+Yes, the Eventing Service node can be co-located with other MDS-enabled services.
+
+
+* Is it supported for both text and binary formats of the documents?
++
+The value or the document must always be text in the form of a JSON document.
+
+
+== *Change Capture*
+
 * Will all the updates to a document be seen in DCP?
 +
 When a document is updated multiple times in a small time-window, not all updates are seen in DCP.
 This means that the event handlers see only those deduplicated events that are published in DCP.
 
-* Is it supported for both text and binary formats of the documents?
+
+* Can a Function listen to all changes across Couchbase Server?
 +
-The value or the document must always be text in the form of a JSON document.
+A defined Function listens only to changes that are exposed using the DCP for the buckets (Couchbase and Ephemeral) in the data-nodes.
+Memcached buckets are not supported.
+The Function cannot listen to changes happening in other Couchbase components, such as FTS, Views, or GSI; nor can it listen to system events.
+
 
 * Can we determine what has changed on the document?
 +
-No.
-But you can implement versioning as part of the application logic.
+No. But you can implement versioning as part of the application logic.
 If versioning is important, include it as part of the data architecture.
 
-* What kind of node do I run my Eventing Service on?
+
+* Can I get old and new values of the document inside the Function?
+
 +
-Eventing leverages the latest trends in multi-core CPUs; therefore nodes selected for Eventing should optimally have a higher number of cores than those for indexing.
+No. We do not support versioning of documents; therefore, this feature is not available out of the box. Though customers can have another ‘Mother’ bucket that stores documents that could be looked up, in order to determine the difference between the current document and the last modified.
```

Comment:
Do we define what a "Mother" bucket somewhere else? Also, need to rewrite the sentence for second person.